### ISSUE TITLE
Add term squared off shoulders

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10369]
**Preferred term label:**
Squared off shoulders

**Synonyms**
square shoulders

**Definition (free text, please give PubMed ID)**
Squared looking shoulders based on prominent angle (approximately 90 degrees) between neck and shoulder (neck-trapezius angle) and abnormally less slope of the shouders, looking more like a straight line.
For example:
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/ada197cf-1479-4314-8c30-0950669c0be5)
From PMID33258289

**Parent term (use hpo.jax.org/app)**
Abnormal shoulder morphology HP:0003043

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
THREE M SYNDROME 1; 3M1, OMIM273750 (PMID33258289, PMID30622233)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000464
- label: Squared off shoulders
- parents: [Abnormal shoulder morphology (HP:0003043)](https://hpo.jax.org/app/browse/term/HP:0003043)
- pmid: PMID:33258289
- def: An appearance of the shoulders whereby the neck-trapezius angle (angle between the neck and the shoulder) and the angle formed by the shoulder and the arms hanging at the side of the body are both close to 90 degrees. This gives the should the appearance of the corner of a square.
- comment: See Figure 1 of PMID:33258289.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10369
- synonyms:
